Alicerce/Sources/Logging/Log+ConsoleProvider.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Log+ConsoleProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class ConsoleProvider: LogProvider {
+
+        public enum ConsoleOutput {
```

Comment:
I took some ideas out of SwiftyBeaver, which supports ```NSLog``` as well as ```print```. We surely not need NSLog, and I guess if someone is doing a benchmark to either use Alicerce or SwiftyBeamer, this distinction won't wight too much. I'll remove ```NSLog``` support.